x-pack/solutions/chat/plugins/workchat-app/server/data_types/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DataTypeDefinition } from '@kbn/onechat-plugin/server/services/data';
+import type { OnechatPluginSetup } from '@kbn/onechat-plugin/server';
+
+export const workChatDataTypes: DataTypeDefinition[] = [
```

Comment:
Good that you included example! Can you also registed some dummy UI components from `/public`? This will help you write up the component registry correcly in the plugin definition

Reply:
Good shout, added as part of [9f4dc3d](https://github.com/elastic/kibana/pull/231653/commits/9f4dc3daf6df58f8d35ef235f818674e18ae3e8f)